orgs/altinn-orgs.json

```diff
@@ -227,6 +227,19 @@
                 "production"
             ]
         },
+        "stami": {
+            "name": {
+                "en": "Statens arbeidsmiljøinstitutt",
```

Comment:
Should be "The National Institute of Occupational Health"

Reply:
Fixed